CHANGES.rst

```diff
@@ -25,6 +25,7 @@ New Features
 - ``astropy.io.votable``
 
 - ``astropy.modeling``
+  - Added ``SmoothlyBrokenPowerLaw1D`` model [#5656].
```

Comment:
A really tiny nit-pick but could you move the period (full stop) to the end of the sentence, just after "model" (in case there are automatic tools that read this file).